bfcfg

```diff
@@ -829,7 +829,7 @@ sys_cfg()
   sys_cfg_multi_byte ${tmp_file} "LARGE_ICMC_SIZE" 39 4 "${SYS_LARGE_ICMC_SIZE}"
   sys_cfg_multi_byte ${tmp_file} "CE_THRESHOLD" 45 4 "${SYS_CE_THRESHOLD}"
   sys_cfg_one_byte ${tmp_file} "DISABLE_HEST" 49 "${SYS_DISABLE_HEST}"
-  sys_cfg_multi_byte ${tmp_file} "L3_CACHE_PART_LEVEL" 50 2 "${SYS_L3_CACHE_PART_LEVEL}"
+  sys_cfg_one_byte ${tmp_file} "L3_CACHE_PART_LEVEL" 50 "${SYS_L3_CACHE_PART_LEVEL}"
```

Comment:
Could you please rename it to 'SYS_L3_CACHE_PARTITION_LEVEL' -- I think this is a rebase issue. Chris has introduced this line and and the name isn't consistent with the confluence page.

By the way, this logic is responsible of updating the value at runtime. Have you verified the capsule path?